test/metabase/driver/postgres_test.clj

```diff
@@ -234,8 +234,23 @@
                  (mt/run-mbql-query people
                    {:fields [$name $bird_id->birds.name]}))))))))
 
-(defn- default-table-result [table-name]
-  {:name table-name, :schema "public", :description nil})
+(defn- default-table-result
+  ([table-name]
+   (default-table-result table-name {}))
+  ([table-name opts]
+   (merge
+    {:name table-name :schema "public" :description nil
+     ;; row-count are estimated, so the value can't be know for sure "during" test without
```

Comment:
grammar nit: 

row-count *is* estimated, so the value can't be *known* for sure .. So for tests that *don't* ~~concern~~ require ...

Reply:
updated in https://github.com/metabase/metabase/pull/39494/commits/1ef0ec58be8a5ae875ec3c0a42cb13d219d6b94e. Thanks!